Model/Api/GetProduct.php

```diff
@@ -165,6 +174,22 @@ private function getProductFamily(){
                 array_push($childrenStockArray, $childProductInventory);
             }
             $this->productData->setChildren($childrenStockArray);
+
+            $attributes = $parentProduct->getTypeInstance()->getConfigurableOptions($parentProduct);
```

Comment:
@danac-gs Could you please take a look? Because we are going to use this API very intensively for Pinterest looks like we need to double-check if this is the best way to get all configurable options